src/components/manager/ManagerDropBoxContent.js

```diff
@@ -307,65 +523,91 @@ const ManagerDropBoxContent = () => {
         showFileContentsModal();
     }, []);
 
-    const workflowsSupported = [];
+    const workflowsSupported = useMemo(
+        () => ingestionWorkflows.filter(w => getWorkflowFit(w)[0]),
+        [ingestionWorkflows]);
+
     const workflowMenu = (
         <Menu>
-            {ingestionWorkflows.map(w => {
-                const workflowSupported = getWorkflowFit(w)[0];
-                if (workflowSupported) workflowsSupported.push(w);
-                return (
-                    <Menu.Item key={w.id}
-                               disabled={!workflowSupported}
-                               onClick={() => showTableSelectionModal(w)}>
-                        Ingest with Workflow &ldquo;{w.name}&rdquo;
-                    </Menu.Item>
-                );
-            })}
+            {ingestionWorkflows.map(w => (
+                <Menu.Item key={w.id}
+                           disabled={workflowsSupported.findIndex(w2 => w2.id === w.id) === -1}
+                           onClick={() => showTableSelectionModal(w)}>
+                    Ingest with Workflow &ldquo;{w.name}&rdquo;
+                </Menu.Item>
+            ))}
         </Menu>
     );
 
-    const selectedFileViewable = selectedFiles.length === 1 &&
-        VIEWABLE_FILE_EXTENSIONS.filter(e => selectedFiles[0].endsWith(e)).length > 0;
+    const handleIngest = useCallback(() => {
+        if (workflowsSupported.length !== 1) return;
+        showTableSelectionModal(workflowsSupported[0]);
+    }, [workflowsSupported]);
```

Comment:
On my end the ìngest`button is always disabled

Reply:
not the case for me...